Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -356,6 +358,22 @@ defineSuite([
         });
     });
 
+    it('loads tileset with extras', function() {
+        return Cesium3DTilesTester.loadTileset(scene, tilesetUrl).then(function(tileset) {
+            expect(tileset.extras).toBeDefined();
```

Comment:
CC https://github.com/AnalyticalGraphicsInc/cesium/pull/6974#discussion_r213802589

Checking that the extras is what we expect is still worthwhile:

`expect(tileset.extras).toEqual({'key': 'value'});`

And remove the `toBeDefined` line.